components/background/BackgroundImage.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2006-present TrueChoice IP Holding Company, Inc.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of TrueChoice IP Holding Company, Inc.
+ * ("Confidential Information").  You shall not disclose such Confidential Information and shall
+ * use it only in accordance with the terms of the license agreement you entered into with the company.
+ */
+
+// dependencies
+// local files
+import styles from './BackgroundImage.module.css';
+
+/**
+ * @description - full viewport width/height background image
+ * @param {string} props.bgImageUrl - Image url to display
+ * @return {jsx} - the Background Cover Image to render
+ */
+
```

Comment:
remove extra vertical space between function description comment and function declaration

Reply:
Done